pgcli/key_bindings.py

```diff
@@ -42,7 +42,11 @@ def _(event):
         Force autocompletion at cursor.
         """
         _logger.debug('Detected <Tab> key.')
-        event.cli.current_buffer.complete_next()
+        b = event.cli.current_buffer
+        if b.complete_state:
```

Comment:
Is this detecting if the completion menu is currently showing? 

If so, can I use this to overload the <enter> key? 

One of the outstanding requests is to use the <enter> key to select the current suggestion. Right now the <enter> key finishes the statement and sends it to the server. I'm wondering if that can be overridden if the completion menu showing.